src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+const ADD_BOOK = 'react-bookstore/books/addBook';
+const REMOVE_BOOK = 'react-bookstore/books/removeBook';
```

Comment:
- Please, kindly remove the `react` from `react-bookstore` to meet the project requirement. I know it still serve the same purpose, keeping things simple really helps a lot. :pray: 

```
const ADD_BOOK = 'bookstore/books/addBook';
const REMOVE_BOOK = 'bookstore/books/removeBook';

```